### PR TITLE
Updates for week 9 to 12.

### PR DESCRIPTION
**Pull request by Alt-Shivam**

### CODE REVIEW

**Comment by Alt-Shivam** [comment]
Hi @MSanKeys963 ,
Do let me know, if you have some suggestions.
Thanks,

File: 2022/codec_registry_shivank/README.md [hunk 1]
```diff
@@ -94,7 +94,17 @@ and there documentation in research file.
 * Defined a CodecID Naming Convention for codecs without codecIDs. [Link](https://alt-shivam.github.io/Codecs-Registry/Others/CodecID_Naming_Convention.html)
 
 ## Week 9 & 10
+* Created a python script `main.py` for registry to perform the following operations:
+- Search a codec in remote registry.
+- If codec is not present in remote registry, script will redirect the user to the [link].(https://alt-shivam.github.io/Codecs-Registry/Others/AddNewCodec.html)
+- If codec is present in remote registry, script will display inforamtion `CodecID`,`CodecDescription`,`LastUpdate`,`Type` and `CodecResearch` file link.
```
Comment (sanketverma1704) [inline 2022/codec_registry_shivank/README.md]: There's a link missing here.

File: 2022/codec_registry_shivank/README.md [hunk 2]
```diff
@@ -94,7 +94,17 @@ and there documentation in research file.
...
 
 ## Week 11 & 12
+* Created `Example.json` file to store the examples of codecs. [Link]()
```
Comment (sanketverma1704) [inline 2022/codec_registry_shivank/README.md]: Missing link here.

**Comment by sanketverma1704** [comment]
Hi @Alt-Shivam. Thanks for sending the PR.
I've added a couple of comments above. Please have a look.

File: 2022/codec_registry_shivank/README.md (hunk 1 above)
Comment (Alt-Shivam) [reply 2022/codec_registry_shivank/README.md]: Fixed.

File: 2022/codec_registry_shivank/README.md (hunk 2 above)
Comment (Alt-Shivam) [reply 2022/codec_registry_shivank/README.md]: Fixed.

**Comment by Alt-Shivam** [comment]
Thanks for the comments.
I just resolved them.

**Review by sanketverma1704 — APPROVED** [review]

**Pull request merged**